data/presets/emergency/defibrillator.json

```diff
@@ -4,11 +4,13 @@
         "indoor",
         "ref",
         "operator",
-        "defibrillator/location"
+        "defibrillator/location",
+        "opening_hours",
+        "access",
+        "description"
```

Comment:
I don't have a strong opinion on this, but given that `description` is more of a situation-specific type of information (it should be provided only when there is something special to be disclosed), I would recommend keeping it in 'moreField'. Overall I agree that this field is helpful.

Reply:
Hi there.
I don't mind to change it. The PR is never accepted anyway and i think, never will.

Reply:
Please address the comments and I will work on accepting the change :)